pages/docs/project-management/drafts.mdx

```diff
@@ -0,0 +1,48 @@
+---
+description: Hashboard drafts allow you to stage changes to your project safely and apply them in a single operation when you're ready. 
+---
+
+# Drafts
+
+Drafts are a way to stage changes to your Hashboard project safely. 
+Drafted resources can be created, updated, and deleted, and can be applied to your project in a single operation. 
+
+## Creating a draft
+Drafts live in the [Drafts page](https://hashboard.com/app/p/drafts) within a project. 
+
+To create a draft, click the `New draft` button. You'll be prompted for a name and description, which can be changed later.
+
+Once you've created a draft, you can enter the draft by clicking on the `Start drafting` button. 
+
+While in a draft, you should see an orange `Drafting` button at the top of the page to indicate that you're in a draft. 
+This means you're safe to interact with the project and make changes to resources, knowing that they will not be applied to your project until you're ready.
+
+To exit the draft, click the `X` button on the orange `Drafting` button at the top of the page, or navigate back to the Drafts page.
```

Comment:
Could mention there are two buttons here? X leaves you on the current resource just not in draft and the "Drafting" button takes you back to the draft details page.